src/renderer/display_list.rs

```diff
@@ -1149,7 +1149,7 @@ fn format_header<'a>(
                 ..
             } = item
             {
-                if main_range >= range.0 && main_range <= range.1 + end_line.len() {
+                if (range.0..range.1 + end_line.len()).contains(&main_range) {
```

Comment:
This change makes it so highlighting EOF shows the incorrect line-column. I believe the solution to this is to modify the original code to check that `main_range` is less than `range.1 + end_line` not less than or equal, as well as make it so `EOF` adds 1.
```suggestion
                if main_range >= range.0 && main_range < range.1 + max(*end_line as usize, 1) {
```

Reply:
That seems to work, thanks for the suggestion!